source/api/image/3.0/index.md

```diff
@@ -395,6 +395,8 @@ The `default` quality exists to support [level 0 compliant implementations][imag
 
 Regardless of level of compliance, services that make additional qualities beyond `default` available _MUST_ list those qualities in the [`extraQualities` property][image3-extra-functionality] of the response to the [Image Information Request][image3-information]. A request for an image with an unsupported quality value _SHOULD_ return a 400 (Bad Request) status code.
 
+**N.B.** A request for the `color` quality of an image may return an image comprised solely of grayscale or bitonal pixels if this is all of the color information the server has available. The server _SHOULD NOT_ include `color` in the `extraQualities` list when this is a case, but a request of this nature _MUST NOT_ result in an error result in an error.
```

Comment:
"result in an error" is repeated twice at the end of the paragraph.

Reply:
Oops, thanks. Fixed.